src/bootman/kernel.c

```diff
@@ -140,6 +166,14 @@ Kernel *boot_manager_inspect_kernel(BootManager *self, char *path)
                 kern->kconfig_file = strdup(kconfig_file);
         }
 
+        if (nc_file_exists(initrd_file)) {
+                kern->initrd_file = strdup(initrd_file);
```

Comment:
I can chuck in an OOM check, I think at the time I was just being consistent with the other kconfig one